index.html

```diff
@@ -170,6 +170,41 @@ <h4>LET’S CONNECT</h4>
                <span><h3>Frameworks</h3><img src="images/right-button.png"></span> <hr>
                <span><h3>Skills</h3><img src="images/right-button.png"></span> <hr>
     </section>
-    </div>
+   
+    <footer class="contact-form">
+      <div class="text-container">
+        <h2 class="primary-text">Contact me</h2>
+        <p class="paragraph-details">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/xyyvbnal" method="post">
+        <input
+          type="text"
+          placeholder="Enter name..."
+          required
+          maxlength="30"
+          name="user_name"
+        />
+        <input
+          type="email"
+          placeholder="email@example.com"
+          name="user_email"
+          required
+        />
+        <textarea
+          cols="30"
+          rows="10"
+          maxlength="500"
+          name="user_message"
+          placeholder="Write your message here"
+          required
+        ></textarea>
+        <button type="submit" class="form-btn">Get in Touch</button>
```

Comment:
[ Required ] Please consider aligning the `Get In Touch`  button to the left as illustrated in the Figma template, the button is currently centered which is not in line with the Figma template given.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/199624619-101a5736-9e46-4871-a241-1d929e787261.png" width="300px" />

## Your implementation of the design
<img src="https://user-images.githubusercontent.com/16392046/199624711-405c0b9f-73aa-4e00-877d-e7c8cb24b1b4.png" width="300px" />